index.html

```diff
@@ -11,250 +11,339 @@
-          <li class="nav-item">
-            <a href="#"><i class="fas fa-bars"></i></a>
-          </li>
-        </ul>
-      </nav>
-    </header>
-    <section class="hero-container">
-      <div class="sub-container">
-        <h1 class="hero-highlight-1">
-          Hey There. <br />
-          I’m OMSOO
-        </h1>
-        <h3 class="hero-highlight-2">I’am a Software Developer</h3>
-        <p class="hero-highlight-3">
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
-        <ul class="social-icons">
-          <li>
-            <a href="#"><i class="fab fa-linkedin-in"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fab fa-twitter"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fab fa-angellist"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fab fa-pinterest-p"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fab fa-github"></i></a>
-          </li>
-        </ul>
-      </div>
-    </section>
-    <div class="recent-work">
-      <p class="resent-work-txt">my recent works</p>
-      <hr />
-    </div>
-    <section class="grid-container">
-      <article class="cd-main-header">
-        <div class="placeholder-img"></div>
-        <div class="placeholder-content">
-          <h2 class="placeholder-header-txt">multi-post stories</h2>
-
-          <p class="place-holder-txt">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
+  <body>
+    <div class="main-container">
+      <header>
+        <div class="navbar">
+          <a href="#">OMSOO</a>
+          <div class="topnav-right">
+            <a href="#" class="nv-hide">Portfolio</a>
+            <a href="#" class="nv-hide">About</a>
+            <a href="#" class="nv-hide">Contact</a>
+            <div class="nv-hide">
+              <a href="#"><i class="fas fa-envelope"></i></a>
+            </div>
+            <div class="fa-barrs">
+              <a href="#"><i class="fas fa-bars"></i></a>
+            </div>
+          </div>
+        </div>
+      </header>
+      <div class="hero-container sub-container">
+        <div class="hero-sctn">
+          <h1 class="hero-highlight-1">
+            Hey There. <br />
+            I’m OMSOO
+          </h1>
+          <h3 class="hero-highlight-2">I’am a Software Developer</h3>
+          <p class="hero-highlight-3">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
```

Comment:
- [ ] Great job so far 👍 your page looks good and it is responsive. It would be great if your background image takes roughly a third of your page. Right now the background image takes more than half the width of your page which is not the case in the template. Kindly address this issue.
- Your design:
![Annotation 2022-02-26 064434](https://user-images.githubusercontent.com/85113567/155831179-f2ca1d95-e8b1-40cb-9571-c35c03e71ffa.jpg)

- Figma design:
![Annotation 2022-02-26 064353](https://user-images.githubusercontent.com/85113567/155831193-bd403b0d-a0e2-40d8-9e47-140fe08e6d06.jpg)

Reply:
resolved 👍